node/image-generation-with-gemini/README.md

```
@@ -0,0 +1,107 @@
+APPWRITE_FUNCTION_API_ENDPOINT=https://cloud.appwrite.io/v1
+APPWRITE_FUNCTION_PROJECT_ID=YOUR_PROJECT_ID
+APPWRITE_FUNCTION_API_KEY=YOUR_API_KEY
+APPWRITE_BUCKET_ID=Generated_Images
+```
+
+
+## ✅ Expected output
+
+On success, you'll see logs like:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)